core/new-gui/src/app/common/service/user/user.service.ts

```diff
@@ -103,10 +104,8 @@ export class UserService {
    * @param user
    */
   public changeUser(user: User | undefined): void {
-    if (this.currentUser !== user) {
-      this.currentUser = user;
-      this.userChangeSubject.next(this.currentUser);
-    }
+    this.currentUser = user;
```

Comment:
why is this check removed? I thought it's always good to do a comparison check to avoid unnecessary events

Reply:
when there is no user logged in (undefined), I want it still yield a value to trigger some events, for example check if one can access the workflow (not logged in of course cannot access any workflow)